pkg/types/types.go

```diff
@@ -7,15 +7,14 @@ import (
 
 // VfState represents the state of the VF
 type VfState struct {
-	HostIFName   string
-	SpoofChk     bool
-	AdminMAC     string
-	EffectiveMAC string
-	Vlan         int
-	VlanQoS      int
-	MinTxRate    int
-	MaxTxRate    int
-	LinkState    uint32
+	HostIFName string
+	SpoofChk   bool
+	AdminMAC   string
```

Comment:
nit: maybe we can change this to just mac now that we only handle the mac address via the PF